src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachToCaseCallbackService.java

```diff
@@ -118,13 +121,19 @@ public AttachToCaseCallbackService(
 
         return getValidation(exceptionRecordDetails, requesterIdamToken, requesterUserId)
             .map(callBackEvent -> tryAttachToCase(callBackEvent, exceptionRecordDetails, ignoreWarnings))
-            .map(attachToCaseResult ->
-                attachToCaseResult.map(modifiedFields ->
-                    mergeCaseFields(exceptionRecordDetails.getData(), modifiedFields))
```

Comment:
`mergeCaseFields` no longer in use?

Reply:
good point, weird that it's not flagged by sonar...

Reply:
i noticed latest IDE update also misses some occasions in flagging up unused code :monocle_face: